pkg/addons/default/kube_proxy_test.go

```diff
@@ -32,12 +32,12 @@ var _ = Describe("KubeProxy", func() {
 			input = da.AddonInput{
 				Region:              "eu-west-1",
 				EKSAPI:              mockProvider.EKS(),
-				ControlPlaneVersion: "1.20.1",
+				ControlPlaneVersion: "1.21.1",
 			}
 
 			mockProvider.MockEKS().On("DescribeAddonVersions", mock.Anything, &awseks.DescribeAddonVersionsInput{
 				AddonName:         aws.String("kube-proxy"),
-				KubernetesVersion: aws.String("1.20"),
+				KubernetesVersion: aws.String("1.21"),
```

Comment:
maybe we could have `var kubernetesVersion = aws.String("1.21")` defined at the very beginning of the test.

Reply:
`1.21` is also used a lot in addon versions and container images, but this is outside the scope of this PR.